src/components/DayPicker.jsx

```diff
@@ -1093,7 +1100,12 @@ DayPicker.propTypes = propTypes;
 DayPicker.defaultProps = defaultProps;
 
 export { DayPicker as PureDayPicker };
-export default withStyles(({ reactDates: { color, font, zIndex } }) => ({
+
+export default withStyles(({
+  reactDates: {
+    color, font, noScrollBarOnVerticalScrollable, spacing, zIndex,
```

Comment:
nit: I'd prefer these to be on their own lines (for ease of rebasing in the future)

Reply:
+1, see https://github.com/airbnb/react-dates/pull/1298#discussion_r207699220